readme.md

```diff
@@ -126,6 +126,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [PuTTY](https://www.putty.org/) - SSH and telnet client, developed originally by Simon Tatham for the Windows platform.
 * [Terminator](https://gnometerminator.blogspot.com/p/introduction.html) - The robot future of terminals.
 * [Terminology](https://github.com/billiob/terminology) - The best terminal emulator based on the Enlightenment Foundation Libraries.
+* [Terminus](https://eugeny.github.io/terminus/) - A cross-platform terminal for a more modern age, based on web technologies.
```

Comment:
Please use the link to the repositories home GitHub page: 'https://github.com/Eugeny/terminus'